eosiojavarpcprovider/build.gradle

```diff
@@ -1,140 +1,67 @@
-apply plugin: 'com.android.library'
-//Remove 'com.jfrog.artifactory' plugin if you are not using Artifactory
-apply plugin: "com.jfrog.artifactory"
-apply plugin: 'maven-publish'
+plugins {
+    id 'java-library'
+    //Remove 'com.jfrog.artifactory' plugin if you are not using Artifactory
+    id 'com.jfrog.artifactory' version '4.10.0'
+    id 'maven-publish'
 
-android {
-    compileSdkVersion 28
-
-    defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
-        versionCode 1
-
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
-
-    lintOptions {
-        abortOnError false
-    }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     api 'one.block:eosiojava:0.1.2'
 
     implementation 'org.jetbrains:annotations:17.0.0'
+    compile 'com.googlecode.json-simple:json-simple:1.1.1'
     api 'com.squareup.retrofit2:retrofit:2.5.0'
     api 'com.squareup.retrofit2:converter-gson:2.5.0'
     api 'com.squareup.okhttp3:logging-interceptor:3.12.1'
 
-    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
-
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.0-alpha4'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0-alpha4'
+    testImplementation 'org.mockito:mockito-core:3.0.0'
```

Comment:
When I drop in this version of the RPC Provider library in the Android example app I get duplicate class errors during the build process.  We may need to see if the dependencies you have declared are declared elsewhere using different version numbers.

Reply:
These warnings came from the `com.googlecode.json-simple` package. Resolution options here are:
- Exclude `org.hamcrest` package subdependency (in this PR)
- Add the following line to a consumer:
```
configurations {
    compile.exclude group: "junit", module: "junit"
}
```